spacy/about.py

```diff
@@ -1,6 +1,6 @@
 # fmt: off
 __title__ = "spacy"
-__version__ = "3.0.0"
+__version__ = "3.0.1"
```

Comment:
```suggestion
__version__ = "3.0.1.dev0"
```

Let's just build a dev version so we can publish it to pypi without stress?